src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
+    
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    public Formatter() { } 
+    
+    /** Formats a list of strings as a viewable indexed list. */
+    public static String getIndexedListForViewing(List<String> listItems) {
+        final StringBuilder formatted = new StringBuilder();
+        int displayIndex = 0 + DISPLAYED_INDEX_OFFSET;
+        for (String listItem : listItems) {
+            formatted.append(getIndexedListItem(displayIndex, listItem)).append("\n");
+            displayIndex++;
+        }
+        return formatted.toString();
+    }
+
```

Comment:
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.